service.go

```diff
@@ -66,6 +67,32 @@ func (s *Service) GetDevice(id int) (*Device, error) {
 	return s.Dao.GetDevice(id)
 }
 
-func (s *Service) GetManyDevices(limit int, page int) ([]Device, error) {
-	return s.Dao.GetManyDevices(limit, page)
+func (s *Service) GetPaginatedDevices(limit int, page int) ([]Device, error) {
+	return s.Dao.GetPaginatedDevices(limit, page)
+}
+
+func (s *Service) StartTickerService() error {
+	devices, err := s.Dao.GetAllDevices()
+	if err != nil {
+		return err
+	}
+
+	mch := make(chan Measurement)
+	stopDevice := make(chan bool)
+	defer close(stopDevice)
+	defer close(mch)
+
+	go s.MeasurementsWriter(mch)
+
+	for i := range devices {
+		go devices[i].deviceTicker(mch, stopDevice)
```

Comment:
hmm I see what you did here.. this might collide with my previous comment. If you have hard time refactoring it let me know